pygluu-compose/pygluu/compose/templates/docker-compose.yml

```diff
@@ -31,16 +31,16 @@ services:
       - VAULT_REDIRECT_INTERFACE=eth0
       - VAULT_CLUSTER_INTERFACE=eth0
       - VAULT_ADDR=http://0.0.0.0:8200
-      - VAULT_LOCAL_CONFIG={"backend":{"consul":{"address":"consul:8500","path":"vault/"}},"listener":{"tcp":{"address":"0.0.0.0:8200","tls_disable":1}}}
+      - VAULT_LOCAL_CONFIG={"backend":{"consul":{"address":"${GLUU_CONSUL_NAME}:8500","path":"vault/"}},"listener":{"tcp":{"address":"0.0.0.0:8200","tls_disable":1}}}
```

Comment:
The address can be pointed to `consul` service, so changing it to container name is unnecessary.